low-level/message_handlers/node_data_msg_handler.py

```diff
@@ -605,6 +631,12 @@ def _generate_if_data(self):
 
         nw_alerts = self._get_nwalert(interfaces)
 
+        # Check if other node is reachable on the pvt data nw
+        # Before that, take a copy of prev value
+        other_node = 'srvnode-2' if node_key_id == 'srvnode-1' else 'srvnode-1'

```

Comment:
shouldn't hardcode, but check from salt db only. Get both node key id from salt db & then check for self node id again, which ever remains is other node